apps/www/content/docs/installation/laravel.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: Laravel
+description: Install and configure Laravel with Inertia
+---
+
+<Steps>
+
+### Create project
+
+Start by creating a new Laravel project with Inertia and React using the laravel installer `laravel new my-app`:
+
+```bash
+laravel new my-app --typescript --breeze --stack=react --git --no-interaction
+```
+### Run the CLI
+
+Run the `shadcn-ui` init command to setup your project:
+
+```bash
+npx shadcn-ui@latest init
+```
+
+### Configure components.json
+
+You will be asked a few questions to configure `components.json`:
+
+```txt showLineNumbers
+Would you like to use TypeScript (recommended)? no / yes
+Which style would you like to use? › Default
+Which color would you like to use as base color? › Slate
+Where is your global CSS file? › resources/app.css
+Do you want to use CSS variables for colors? › no / yes
+Where is your tailwind.config.js located? › tailwind.config.js
+Configure the import alias for components: › @/components
+Configure the import alias for utils: › @/lib/utils
```

Comment:
This should be:
```bash
@/Lib/utils
```

Reply:
Thank you for the valuable feedback. I've made the changes suggested, but chose to retain the `lib` folder name as lowercase.